lib/talkingscore.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
     <meta name="robots" content="noindex" />
     
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
```

Comment:
I'd highly recommend taking a look at https://htmx.org for interactivity.  It plays very nicely with Django.

Reply:
I probably don't need to use jquery here.  I think I am only using it to find document elements!  I shall have a look at htmx

Reply:
Fixed in 3329eb4 and 6816528
Might have reduced compatibility with older browsers?
I didn't think adding HTMX seemed necessary for what we were currently doing with jQuery?